code/ui/manager/src/components/mobile/navigation/MobileAddonsDrawer.tsx

```diff
@@ -1,59 +1,22 @@
 import type { FC, ReactNode } from 'react';
-import React, { useRef } from 'react';
+import React from 'react';
 import { styled } from '@storybook/theming';
-import { Transition } from 'react-transition-group';
-import type { TransitionStatus } from 'react-transition-group/Transition';
-import { useLayout } from '../../layout/LayoutProvider';
 
 interface MobileAddonsDrawerProps {
   children: ReactNode;
 }
 
-const TRANSITION_DURATION = 200;
-
-const Container = styled.div<{ state: TransitionStatus }>(({ theme, state }) => ({
-  position: 'fixed',
+const Container = styled.div(({ theme }) => ({
   boxSizing: 'border-box',
   width: '100%',
   background: theme.background.content,
-  height: '50%',
-  bottom: 0,
-  left: 0,
+  height: '42vh',
   zIndex: 11,
-  transition: `all ${TRANSITION_DURATION}ms ease-in-out`,
+  transition: 'all 200ms ease-in-out',
```

Comment:
does this transition still have an impact? if not we should remove it.

Reply:
I removed it.